scripts/cli_create_kb.py

```diff
@@ -0,0 +1,43 @@
+"""Functionality for creating the knowledge base from downloaded assets and by querying Wikipedia's API."""
+import logging
+import os
+from pathlib import Path
+
+import spacy
+import typer
+
+from extraction import get_paths
+from kb import WikiKB
+
+
+def main(vectors_model: str, language: str):
+    """Create the Knowledge Base in spaCy and write it to file.
+    language (str): Language.
+    vectors_model (str): Name of model with word vectors to use.
+    """
+    logger = logging.getLogger(__name__)
+    logger.info("Constructing knowledge base.")
+
+    nlp = spacy.load(vectors_model, exclude=["tagger", "lemmatizer", "attribute_ruler"])
```

Comment:
Mh, I currently use this pipeline in the benchmark for NER. This might not be necessary - after all, when running the benchmark/any other project, I can just instantiate the pipeline twice instead. 

So right now I need everything required to embed and run NER (so `tok2vec`, `ner`, `senter`/`sentencizer`?). If we want to drop persisting the pipeline used here for later reuse (?), then I guess I'd only need `tok2vec`.

Reply:
After internal discussions I'll leave this is as is, as the next PR will eliminate the need for all components* due to incorporating entities and their span offsets directly from corpus annotations. 

*Except maybe `senter`/`sentencizer`? I seem to remember that the NEL model accesses sentence information in the docs with `incl_context is True`.